specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/hybridRunbookWorker.json

```diff
@@ -0,0 +1,469 @@
+            "required": true,
+            "type": "string",
+            "description": "The hybrid runbook worker group name"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HybridRunbookWorkersListResult"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/HybridRunbookWorker"
+      }
+    }
+  },
+  "definitions": {
+    "HybridRunbookWorker": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/HybridRunbookWorkerProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the hybrid worker group properties."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "Resource system metadata."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../common/v1/definitions.json#/definitions/Resource"
+        }
+      ],
+      "description": "Definition of hybrid runbook worker."
+    },
+    "HybridRunbookWorkerProperties": {
+      "type": "object",
+      "properties": {
+        "ip": {
+          "type": "string",
+          "description": "Gets or sets the assigned machine IP address."
+        },
+        "registeredDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the registration time of the worker machine.",
+          "x-nullable": false
+        },
+        "lastSeenDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last Heartbeat from the Worker",
+          "x-nullable": false
+        },
+        "vmResourceId": {
+          "type": "string",
+          "description": "Azure Resource Manager Id for a virtual machine."
+        },
+        "workerType": {
+          "type": "string",
+          "description": "Type of the HybridWorker.",
+          "enum": [
+            "HybridV1",
+            "HybridV2"
+          ],
+          "x-ms-enum": {
+            "name": "WorkerTypeEnum",
```

Comment:
>WorkerTypeEnum [](http://example.com/codeflow?start=21&length=14)

Use just the name 'WorkerType'. This name is used to generate C#/Java SDK classes.

Reply:
Made the change requested.